backend/postgres/init.sql

```diff
@@ -2,7 +2,7 @@ GRANT ALL PRIVILEGES ON DATABASE docker TO docker;
 
 CREATE TABLE benchmarks(
 	repositories varchar(256),
-	commits varchar(50) NOT NULL UNIQUE,
+	commits varchar(50) NOT NULL,
```

Comment:
If repo+commit is required to be unique, it may also be useful to have a UNIQUE(repositories, commits). Should branch also be included in this?

Reply:
My suggestion is that we don't actually want to have a unique constraint on repo+commit columns.

It is IMO useful to be able to re-run the benchmarks for the same commit multiple times to increase confidence in the results. The repeated runs could be used to calculate more advanced stats. For now just showing the repeated runs on the graph with the same commit label isn't very problematic.

I plan on improving the database schema (to remove the duplication between `benchmarks` and `benchmarksrun`) in future PRs where I can look into this more carefully.

Reply:
Yes, that makes sense - thanks